spec/features/visitor_can_view_items_spec.rb

```diff
@@ -1,28 +1,30 @@
 require "rails_helper"
 
-feature "View items" do
-  let!(:item_one) {Item.create(name: "Canon 7d",
-                                description: "The best camera ever (NOT)",
-                                price: 15)}
+RSpec.describe "VisitorCanViewItems", type: :feature do
+  feature "View items" do
+    let!(:item_one) {Item.create(name: "Canon 7d",
+                                  description: "The best camera ever (NOT)",
+                                  price: 15)}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 7, 45 should be on its own line.